### DISCUSSION TITLE
Function Events

### DISCUSSION THREAD

**Discussion by Mysterious-Madness** [YoshiCrafter29/YoshiCrafterEngine#133] [Q&A]
So, I am making a mod with Yoshi engine (basically testing before I make it)
And I've been trying to make events, but it has no instructions, and my small brain doesn't comprehend it
(Like what does parameters event mean in this engine)

**Comment by BernardoGP4504** [comment]
parameters are what you have to fill and to make your event type:
`function eventname(Param1:String, Param2:String...)`
Param[number that i put] can be whatever the user needs to fill and the '...' means you can add as many as you want (also you can name them whatever you like but the '`:String`' part has to be included)
eventname can be whatever name you want the event to be named and you can have as many events in one file as you want
